docs/recipes/virtualenv.rst

```diff
@@ -6,22 +6,21 @@ that uses SCons to build.  A virtualenv is a way to create an
 isolated execution environment - you can install whatever you need
 there, and change versions as needed, without affecting anything
 else on your system that uses Python.  Here is an example session
-for setting one up:
+for setting one up on a Linux host:
 
 .. code:: shell
 
    $ cd Work
    $ python -m venv myvenv
-   $ cd myvenv
    $ source bin/activate
```

Comment:
`source myvenv/bin/activate`

Reply:
and actually `. myvenv/bin/activate` should also work. (that's what I typically do, less typing.. )

Reply:
yeah, hand-editing logs never works out right

Reply:
I can trim these further if you want - just trying to figure out what still needs doing here to complete this.

Reply:
Yup. If you can fix that, then we're good to go.